pkg/proxy/ipvs/proxier.go

```diff
@@ -899,6 +899,34 @@ func (proxier *Proxier) OnEndpointSlicesSynced() {
 // EntryInvalidErr indicates if an ipset entry is invalid or not
 const EntryInvalidErr = "error adding entry %s to ipset %s"
 
+func getLocalAddrs() ([]net.IP, error) {
+	var localAddrs []net.IP
+
+	addrs, err := net.InterfaceAddrs()
+	if err != nil {
+		return localAddrs, err
+	}
+
+	for _, addr := range addrs {
+		ip, _, err := net.ParseCIDR(addr.String())
+		if err != nil {
+			klog.Errorf("parse cidr failed when getting local addresses, err: %v", err)
```

Comment:
This should still return error IMO

Reply:
If there is 100 adds, 1 of them parsed error, we could still get 99 correctly. If return error directly, we may get empty local addrs, so any external ip is assuming not local ip. How do you think about this?

Reply:
I think we can consider not returning err here but should be in another PR.